kubernetes/update_images.py

```diff
@@ -2,15 +2,31 @@
 
 import json
 import subprocess
+from argparse import ArgumentParser
 
 import docker
 from docker.models.images import Image
 
+IMAGE_TYPES = ["cpp", "python"]
```

Comment:
Very much a nitpick but I lean towards using immutable types like tuples by default to prevent accidental mutation later on.
```suggestion
IMAGE_TYPES = ("cpp", "python")
```